sql/driver/conn.go

```diff
@@ -186,6 +188,10 @@ func (c *conn) Send(call sqlwire.Call) (*rows, error) {
 			} else if datum.StringVal != nil {
 				t[j] = *datum.StringVal
 			}
+			if t[j] != nil && !driver.IsValue(t[j]) {
```

Comment:
I want you to use `IsScanValue` because that is what the `database/sql/driver` interface is supposed to be guaranteeing here. 

FYI, you don't need the `t[j] != nil` check. See https://golang.org/src/database/sql/driver/types.go?s=5072:5104#L166.